config/sources/families/media.conf

```diff
@@ -78,10 +82,9 @@ case $BRANCH in
 		else
 			KERNELPATCHDIR='media-'$BRANCH
 			LINUXFAMILY=media
-
-			if [[ $BOARD == jetson-nano ]]; then
-				MODULES_INITRD="jetson-nano-current"
-			fi
+#			if [[ $BOARD == jetson-nano ]]; then
+#				MODULES_INITRD="jetson-nano-current"
+#			fi
```

Comment:
Yeah, gotcha, no need to comment-out, just remove. `git` has the history...

Reply:
> Yeah, gotcha, no need to comment-out, just remove. `git` has the history...

After comprehensive testing, this will be removed, but until then, let it be.

Reply:
https://git-scm.com/book/ru/v2

Reply:
> https://git-scm.com/book/ru/v2

Have you decided to tell me how to work with GIT? I'm not an expert, but I know enough to evaluate WHY, I don't want to delete this code YET (don't waste time talking about commits and stories) :) If you are struggling so much for the purity of the code, I advise you to look at the shit that is in the kernel patch directories (a bunch of old patch versions from previous kernels). It is absolutely useless, because the build will not work if you try to switch to these old patches.